### PR TITLE
VFB-107: fix lists page error

### PR DESCRIPTION
**Pull request by rickaw**
## What's changed
- Seed the website_data table in the local database to show comments at the top
- Use Supabase-native way to get one row out of the database
- Fix the lists table not showing the quantities and notes

| Before | After |
| --- | --- |
| image | image |

- [x] Make sure you've verified it works on dev via `npm run dev`
- [x] Make sure you've verified it works on prod via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
  - Will also be automatically checked on pull request!
- [x] Make sure you've tested via `npm run test`
  - Will also be automatically checked on pull request!

### CODE REVIEW

File: src/common/fetch.ts [hunk 1]
```diff
@@ -107,9 +107,16 @@ export const fetchLists = async (supabase: Supabase): Promise<Schema["lists"][]>
 };
 
 export const fetchComment = async (supabase: Supabase): Promise<string> => {
-    const { data, error } = await supabase.from("website_data").select().eq("name", "lists_text");
+    const { data, error } = await supabase
+        .from("website_data")
+        .select()
+        .eq("name", "lists_text")
+        .limit(1)
+        .single();
```
Comment (stonelink) [inline src/common/fetch.ts:115]: Ah great, so it will actually recognise the error now

File: src/app/lists/ListDataview.tsx [hunk 2]
```diff
@@ -102,8 +102,8 @@ const ListsDataView: React.FC<Props> = (props) => {
                     .map(([key]) => [
                         key,
                         {
-                            quantity: row[`${key}_quantity` as keyof Schema["lists"]],
-                            notes: row[`${key}_notes` as keyof Schema["lists"]],
+                            quantity: row[`quantity_for_${key}` as keyof Schema["lists"]],
```
Comment (stonelink) [inline src/app/lists/ListDataview.tsx:105]: Woops. I should have clocked this in the last review

File: src/common/fetch.ts (hunk 1 above)
Comment (stonelink) [reply src/common/fetch.ts:115]: Btw [VFB-57](https://softwiretech.atlassian.net/browse/VFB-57) is about the fact that nothing useful happens with the subsequent DatabaseError

File: src/app/lists/ListDataview.tsx (hunk 2 above)
Comment (rickaw) [reply src/app/lists/ListDataview.tsx:105]: Completely my fault for not checking that table on the page :)

File: src/common/fetch.ts (hunk 1 above)
Comment (rickaw) [reply src/common/fetch.ts:115]: Thanks for the info!

**Pull request merged**